src/integration-tests/test_admin_res_log.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2024 Bloomberg Finance L.P.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This suite of test cases verifies that only the node that the admin
+connects to send the response back to the admin, and the corresponding
+repsonse is logged.
+"""
+
+import blazingmq.dev.it.testconstants as tc
+from blazingmq.dev.it.fixtures import (
+    multi_node,
+    Cluster,
+)
+from blazingmq.dev.it.process.admin import AdminClient
+from blazingmq.dev.it.process.client import Client
+
+TIMEOUT = 2
+
+
+def test_adminsession_res_log_stat(multi_node: Cluster):
+    """
+    Test: STAT SHOW command: response message only generate on the node itself
+    """
+
+    admin = AdminClient()
+
+    # find the first two nodes which are not the known leader
+    member1 = next(
```

Comment:
For here and below, could it make sense to do something like

```
replicas = multi_node.nodes(exclude=node.last_known_leader)
```

and use replicas[0]/replicas[1] below?